test_requirements.txt

```diff
@@ -3,3 +3,4 @@ cram>=0.6
 mock
 coverage
 coveralls
+unittest2; python_version == '2.6'
```

Comment:
Cool, I didn't know one could do that. Might even be able to remove the `mock` dependency for `unittest.mock` on Py35